src/app/containers/Timestamp/index.jsx

```diff
@@ -13,18 +13,31 @@ import {
 } from './timestampUtilities';
 import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 
-const isToday = timestamp => {
-  const today = moment(Date.now());
-  return today.isSame(timestamp, 'day');
+const isSameDay = (dayToCompare, timestamp) => {
+  const day = moment(dayToCompare);
+  return day.isSame(timestamp, 'day');
 };
 
-const formatType = timestamp =>
-  isToday(timestamp) ? formatDateAndTime : formatDate;
+const isToday = timestamp => isSameDay(Date.now(), timestamp);
 
-const humanReadable = ({ timestamp, shouldMakeRelative }) =>
+const formatType = timestamps => {
+  if (isToday(timestamps[0])) {
+    if (timestamps.length > 1 && !isSameDay(timestamps[1], timestamps[0])) {
+      return formatDate;
+    }
+    return formatDateAndTime;
+  }
+  return formatDate;
+};
```

Comment:
I've refactored this block to be `if/else` rather than ternary. The ternary was added before the extended requirements added by this ticket. I think this is easier to understand than an alteration to the existing ternary.